### DISCUSSION TITLE
Suggestions welcomed!

### DISCUSSION THREAD

**Discussion by prakkritti1999** [community/community#145390] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

Question

### Body

I am open to any sort of suggestions to my Microservices based project.
[https://github.com/prakkritti1999/Microservices_Demo.git](url)

**Comment by ebndev** [comment]
Hi @prakkritti1999, thanks for sharing with the community! 

Unfortunately, we currently do not allow self-promotion or advertising on the Community Discussions. We want to make sure there is space for users to ask questions without overwhelming them with other conversations. Thank you for helping us maintain a productive and tidy community for all our members.
